submissions/HannaSyn/Popup/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Awesome Popup</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navigation">
+      <ul class="navigation__list">
+        <li class="navigation__link">
+          <a href="#"><img src="assets/mail.svg" alt=""></a>
+        </li>
+        <li class="navigation__link link">
+          <div class="link__menu menu">
+            <input class="menu__input" type="checkbox" id="btn-nav" />
+            <label class="menu__label" for="btn-nav"><img src="assets/menu.svg" /></label>
+            <div class="menu__popup menu-popup invisible">
```

Comment:
The rule `menu__popup ` is empty also? Is there something important in empty classrooms?

Reply:
I can be wrong, but I did it to BEM technology)

Reply:
The class name `menu__popup` says that this `popup` is a child of the parent `menu`, as in the BEM methodology.
Then what does the "menu-popup" class mean?

Reply:
The "menu-popup" class is modifier for next child elements.

Reply:
Ok this is just a discussion))
I would leave only this class name `menu__popup`

Reply:
It's ok!) but I don't understand, how you suggest to name child elements of `menu__popup` according BEM? Maybe, I can't understand this metodology?

Reply:
Maybe I called them `popup__link` 🤔

Reply:
You can nest BEM blocks into other BEM elements. In this case, will be better to "create" BEM block `.popup` simple and short.

Reply:
Thank you for reviews!